### PR TITLE
[Merged by Bors] - fix: Producer and Consumer configuration not incorporated in docker run command

### PR DESCRIPTION
**Pull request by davidbeesley**
I'm somewhat worried about this change as I don't know what it might break with other connectors.

### CODE REVIEW

**Comment by sehz** [comment]
can you rebase? seems like diff is incorrect

**Comment by davidbeesley** [comment]
> can you rebase? seems like diff is incorrect

done

**Comment by sehz** [comment]
clippy failures

File: rust-connectors/common/src/config.rs [hunk 1]
```diff
@@ -89,8 +90,39 @@ impl ConnectorConfig {
...
+        let mut params = Vec::new();
+        if let Some(producer) = self.producer.as_ref() {
+            if let Some(linger) = producer.linger {
+                params.push("--producer-linger".to_string());
+
+                params.push(format!("{}ms", linger.as_millis()));
+            }
+            if let Some(compression) = producer.compression {
+                params.push("--producer-compression".to_string());
+                params.push(format!("{}", compression.to_string()));
```
Comment (morenol) [inline rust-connectors/common/src/config.rs:117]: Looks like we can to this:
```suggestion
                params.push(compression.to_string());
```

Comment (morenol) [reply rust-connectors/common/src/config.rs:117]: Will apply this to see if we can merge this with this change

File: rust-connectors/utils/connector-run/src/main.rs [hunk 2]
```diff
@@ -191,3 +193,45 @@ fn build_args(config: &ConnectorConfig) -> anyhow::Result<Vec<String>> {
...
+  MQTT_URL: mqtt://dummy_url
+producer:
+  batch-size: '10mb'
+  linger: 1s
+  compression: gzip
+consumer:
+  partition: 0
+        "#;
+
+        let config: ConnectorConfig = serde_yaml::from_str(&contents).unwrap();
```
Comment (morenol) [inline rust-connectors/utils/connector-run/src/main.rs:225]:
```suggestion
        let config: ConnectorConfig = serde_yaml::from_str(contents).unwrap();
```

**Review by sehz — APPROVED** [review]

**Pull request closed without merging**